draft-ietf-ace-oscore-profile.xml

```diff
@@ -438,14 +417,97 @@
           </artwork>
         </figure>
 
+        <section anchor="oscore-sec-ctx" title="OSCORE_Security_Context Object">
+
+          <t>
+            A OSCORE_Security_Context is a CBOR map object.  The set of common parameters that can appear in a OSCORE_Security_Context object can be found in the IANA "OSCORE Security Context Parameters" registry (Section <xref target="sec-ctx-params-reg"/>).
+          </t>
+
+          <t>
+            The CDDL grammar describing OSCORE_Security_Context is:
+          </t>
+
+          <figure><artwork type="CDDL"><![CDATA[
+OSCORE_Security_Context = {
+    ? 1 => bstr,              ; ms
+    ? 2 => bstr,              ; clientId
+    ? 3 => bstr,              ; serverId
+    ? 4 => tstr / int,        ; hkdf
+    ? 5 => tstr / int,        ; alg
+    ? 6 => bstr,              ; salt
+    ? 7 => bstr / tstr        ; rpl
+}
+]]></artwork></figure>
+
+          <t>
+            This document defines a set of parameters for an OSCORE_Security_Context object.
+            <xref target="table-key-labels"/> provides a summary of the parameters defined in this section.
+          </t>
+
+          <texttable title="OSCORE_Security_Context Map Labels" anchor="table-key-labels">
+            <ttcol align='left'>name</ttcol>
+            <ttcol align='left'>label</ttcol>
+            <ttcol align='left' width="14em">CBOR type</ttcol>
+            <ttcol align='left'>registry</ttcol>
+            <ttcol align='left'>description</ttcol>
+            
+            <c>ms</c>        <c>1</c>        <c>bstr</c>  <c></c>        <c>OSCORE Master Secret value</c>
```

Comment:
Does the JSON version use this as the label?

Reply:
The JSON version uses "osc" as label of the OSCORE_Security_Context structure (that would appear as a byte string)... was that the question?

Reply:
No, the question is what 'ms' appears as.